lists/downloader_test.go

```diff
@@ -22,27 +21,16 @@ import (
 
 var _ = Describe("Downloader", func() {
 	var (
-		sutConfig                     config.Downloader
-		sut                           *httpDownloader
-		failedDownloadCountEvtChannel chan string
-		loggerHook                    *test.Hook
+		sutConfig  config.Downloader
+		sut        *httpDownloader
+		loggerHook *test.Hook
 	)
 	BeforeEach(func() {
 		var err error
 
 		sutConfig, err = config.WithDefaults[config.Downloader]()
 		Expect(err).Should(Succeed())
 
-		failedDownloadCountEvtChannel = make(chan string, 5)
```

Comment:
now we don't test if metrics counters are being incremented. Maybe we can try to read the counter value and assert it?

Reply:
Yes, that is something we could test for. But I find it not very useful. The Prometheus client `Inc()` type functions are basically a simple Go atomic increment. There's basically no way for it to go wrong in a way that testing would help with.